packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -22,7 +22,9 @@ const { emitter, store } = require(`../redux`)
 const getPublicPath = require(`./get-public-path`)
 const { getNonGatsbyCodeFrameFormatted } = require(`./stack-trace-utils`)
 const { trackBuildError, decorateEvent } = require(`gatsby-telemetry`)
-const { default: errorParser } = require(`./api-runner-node-error-parser`)
+const {
```

Comment:
const { default: errorParser } = require(`./api-runner-error-parser`)

Reply:
that is my prettier :/
try to fix it

Reply:
fixed